### PR TITLE
[*] FO : Added new hooks in authentication

### PR DESCRIPTION
**Pull request by paeddl**
[*] FO : Added new hooks to show images, buttons, text or anything else
below the buttons in the authentication form

### CODE REVIEW

File: controllers/front/AuthController.php [hunk 1]
```diff
@@ -144,6 +144,8 @@ public function initContent()
 
         // Call a hook to display more information on form
         $this->context->smarty->assign(array(
+        	'HOOK_AUTHENTICATE_FORM_BOTTOM' => Hook::exec('displayAuthenticateFormBottom'),
```
Comment (gRoussac) [inline controllers/front/AuthController.php]: Hi,

Thank you for your PR. Are you using tabs or spaces for indentation ?

Regards

Comment (paeddl) [inline controllers/front/AuthController.php]: Hi,
it has been tabs here. I could change it to spaces of course...

Comment (gRoussac) [inline controllers/front/AuthController.php]: yes please !

Comment (paeddl) [inline controllers/front/AuthController.php]: ok, done

**Comment by tchauviere** [comment]
hi @paeddl,

Thanks for your contribution, your PR is now merged !

Best regards,

**Pull request merged**